src/snapred/backend/data/LocalDataService.py

```diff
@@ -1031,28 +1031,35 @@ def initializeState(self, runId: str, useLiteMode: bool, name: str = None):
             peakTailCoefficient=peakTailCoefficient,
         )
 
-        # finally add seedRun, creation date, and a human readable name
-        calibration = Calibration(
-            instrumentState=instrumentState,
-            name=name,
-            seedRun=runId,
-            useLiteMode=useLiteMode,
-            creationDate=datetime.datetime.now(),
-            version=self.VERSION_START,
-        )
+        calibrationReturn = None
+
+        for LiteMode in [True, False]:
```

Comment:
1. use `liteMode`.  `LiteMode` looks like a class name.
2. Does this need to be included in this PR?

Reply:
Done.